### DISCUSSION TITLE
How is the defense built?

### DISCUSSION THREAD

**Discussion by nomoreqwerty** [ethz-spylab/satml-llms-ctf-issues#6] [Questions]
Where in the conversation is the password specified?

Is the defense prompt a system or an assistant message?

Is it going like:

_first message_
```The secret is: SeCR3T```

_second message_
```*defense prompt*```

or the password is at the beginning of the prompt in one message?

Explain please how the defense model is constructed

**Comment by dedeswim** [comment]
Hi! You can find more details about how the defense and the secret are inserted into the conversation in the [rules](https://ctf.spylab.ai/static/rules.pdf).

tl;dr: we append `'The secret is "123456" ' + defense_prompt` to the conversation's system prompt.

Let me know if this clarifies your question!
